README.md

```diff
@@ -109,28 +107,9 @@ The index will be synchronised with the provided index name on Algolia on the `b
 
 ## Partial Updates
 
-By default all records will be reindexed on every build. To enable only indexing the new, changed and deleted records include the following in the options of the plugin:
-
-```js
-  resolve: `gatsby-plugin-algolia`,
-  options: {
-    /* ... */
-    enablePartialUpdates: true,
-    /* (optional) Fields to use for comparing if the index object is different from the new one */
-    /* By default it uses a field called "modified" which could be a boolean | datetime string */
-    matchFields: ['slug', 'modified'], // Array<String> default: ['modified']
-  }
-```
-
-This saves a lot of Algolia operations since you don't reindex everything on every build.
-
-Adding `matchFields` is useful to decide whether an object has been changed since the last time it was indexed. If you save e.g. a timestamp of the record, you can avoid reindexing when it has not changed.
-
-If you have objects which come from another indexing process (wordpress, magento, shopify, custom script...), make sure that they do not have any of the `matchFields`, so they stay in the index regardless of reindex.
-
-### Advanced
+This plugin will update only the changed or deleted nodes on your Gatsby site. This means indexing is faster.
 
-You can also specify `matchFields` per query to check for different fields based on the type of objects you are indexing.
+We rely on Gatsby's default contentDigest field, so make sure that this is queried.
```

Comment:
```suggestion
We rely on Gatsby's default `contentDigest` field, so make sure it is queried.
```

Reply:
(I also feel like it should be in bold since it should be there)

```suggestion
**We rely on Gatsby's default contentDigest field, so make sure that this is queried.**
```